README.md

```diff
@@ -33,6 +33,8 @@ Inside a fresh virtualenv, `cd` into the root folder of this repository (`xblock
 ```bash
 $ pip install -U pip wheel
 $ pip install -r requirements-test.txt
+$ pip install -r $VENV/src/xblock-sdk/requirements/base.txt
+$ pip install -r $VENV/src/xblock-sdk/requirements/test.txt
```

Comment:
Is this really necessary? I tested this several times from scratch to make sure you don't have to install any other requirements besides `pip install -r requirements-test.txt`. The circle CI build only installs `requirements-test.txt` and has been working fine so far.